### PR TITLE
underlines added to Data Depot seperating Shared with me, and published

### PR DESCRIPTION
**Pull request by climberfields**
underlines added to Data Depot seperating Shared with me, and published

### CODE REVIEW

File: designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.js [hunk 1]
```diff
@@ -71,12 +81,36 @@ class DataDepotNavCtrl {
               description: 'Access to my Google Drive for copying'
             }
         );
+
```
Comment (xirdneh) [inline designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.js]: Can you delete this commented code? Thanks.

File: designsafe/static/styles/ng-designsafe.css [hunk 2]
```diff
@@ -29,7 +29,7 @@
     outline: 0;
 }
 
-.ds-data-browser-wrapper {
+.ds-data-browser-wraper {
```
Comment (rstijerina) [inline designsafe/static/styles/ng-designsafe.css]: Shouldn't this be left as `.ds-data-browser-wrapper` (with two 'p's in 'wrapper')?
```suggestion
.ds-data-browser-wrapper {
```

File: designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html [hunk 3]
```diff
@@ -17,11 +17,11 @@
           <a uib-tooltip="{{ childItem.description }}" class="list-group-item" ui-sref="{{ childItem.state }}" ui-sref-active="active">
             {{ childItem.name }}
           </a>
+          <div ng-If="childItem.name == 'Shared with Me'"> <hr> </div>
+          </div>
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html]: Looks like this line doesn't need to be added, while you can keep line #24 to keep the formatting aligned.

File: designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html [hunk 4]
```diff
@@ -17,11 +17,10 @@
           <a uib-tooltip="{{ childItem.description }}" class="list-group-item" ui-sref="{{ childItem.state }}" ui-sref-active="active">
             {{ childItem.name }}
           </a>
+          <div ng-If="childItem.name == 'Shared with Me'"> <hr> </div>
         </div>
-  
       </div>
     </div>
-  </div>
```
Comment (rstijerina) [inline designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html]: Something may have changed, this does not need to be deleted

Comment (rstijerina) [reply designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html]: (line 24)

**Comment by rstijerina** [comment]
Taken care of here:
https://github.com/DesignSafe-CI/portal/pull/490

**Pull request closed without merging**